### PR TITLE
Add KakuteF4V2 tricopter build configuration/target

### PR DESCRIPTION
**Pull request by jihlein**
The KakuteF4V2 is being used on the RCExplorer Tricopters, but the iNav target needs to be custom built for tricopter use.

This PR creates a new target, KAKUTEF4V2TRI, which has the M4 output pad function and the LED pad function swapped.  This allows the tricopter tail servo to be connected to a separate timer from the motor outputs, allowing separate motor/servo output protocols.

The new target hex file has been installed and tested on my RCE TricopterLR.  Motor and servo (connected to LED pad) outputs look correct via logic analyzer trace, and operate correctly as evidenced by correct motor and servo responses (currently using DShot300 for motors with 370 Hz servo).

The LED output (on the M4 pad) looks correct via logic analyzer trace, but I have no LEDs to physically test with.

### CODE REVIEW

File: src/main/target/KAKUTEF4/target.h [hunk 1]
```diff
@@ -147,10 +147,17 @@
 #define ADC_CHANNEL_3_PIN           PC1
 
 #define USE_LED_STRIP
+#if !defined(KAKUTEF4V2TRI)
 #define WS2811_PIN                      PC8
 #define WS2811_DMA_HANDLER_IDENTIFER    DMA2_ST4_HANDLER
 #define WS2811_DMA_STREAM               DMA2_Stream4
 #define WS2811_DMA_CHANNEL              DMA_Channel_7
+#else
```
Comment (digitalentity) [inline src/main/target/KAKUTEF4/target.h:150]: WS2811 defines are no longer necessary

File: src/main/target/KAKUTEF4/target.h [hunk 2]
```diff
@@ -23,14 +23,16 @@
...
-#if defined(KAKUTEF4V2)
+#if defined(KAKUTEF4V2) || defined(KAKUTEF4V2TRI)
 #   define TARGET_BOARD_IDENTIFIER "KTV2"
 #   define USBD_PRODUCT_STRING "KakuteF4-V2"
 #else
 #   define TARGET_BOARD_IDENTIFIER "KTV1"
 #   define USBD_PRODUCT_STRING "KakuteF4-V1"
 #endif
 
+#define USE_TARGET_CONFIG
```
Comment (digitalentity) [inline src/main/target/KAKUTEF4/target.h]: Why this is here?

File: src/main/target/KAKUTEF4/target.h (hunk 1 above)
Comment (jihlein) [reply src/main/target/KAKUTEF4/target.h:150]: Was able to delete the DMA defines, deleting the PIN define led to a compilation error.

File: src/main/target/KAKUTEF4/target.h (hunk 2 above)
Comment (jihlein) [reply src/main/target/KAKUTEF4/target.h]: Not sure why I added that, deleted......

**Comment by jihlein** [comment]
Just flew this configuration using the 280mm tricopter defaults with DShot300 on the motors and 370 Hz servo update rate.  It needs some minor tuning, but overall it flew well.  Rate, attitude, horizon, alt hold, and position hold modes tested.

**Comment by jihlein** [comment]
I don't know what the interest level in this PR is.  I pinged the forum at RCExplorer to gauge interest, as the KakuteF4V2 was being sold as a bundle with their tricopter kits, and some users were at one point asking about iNav use.  We'll see if anyone posts interest here.  Otherwise it can be closed in 2 weeks.

**Comment by Bearingsouth** [comment]
I would like to see this further developed as I am running a KakuteF4V2 on a V4 Tri-copter frame. I 
 would like to apply iNav on my Tri. Thanks Jihlein.

**Comment by ResinSmoker** [comment]
I second this... 

IMO there is a need for Tricopter support for LR (F4V2) users. Especialy considering RCE's (David's) recent announcement that he's soon to close shop.

Now while its possible that Betaflight may be sutible for "some" Tricopter pilots. (Tricopter-Baby, FCF3 Tri's) The RCE-LR tricopters are built for long distance flights that often requires much of the control / navigation features associated with iNav.

Besides, it would be a shame to lose Jilein's work(s) when there are those that would use it.

**Comment by Ivanair3d** [comment]
Hi guys,
I'm currently building a RCExplorer Tricopter with KakuteF4V2 and would be great to fly it with iNav!
Just to support jihlein! 
Cheers!

**Comment by BLeeKiis** [comment]
I would also like to see the servo feedback added as an option to iNav, so much so Ive signed up to help lend weight to the request...
Have recently purchased an LR RCEx' Tricopter frame, had hoped to run a Matech AIO Wing FC & perhaps iNav but will purchase KakuteF4v2 if that is the only option for running all 3 (I hope eventually) flavours of 'Triflight' to compare to each other...
It would also be a shame to loose all the work jhlein & others have so far sunk ito developing the 'Triflight' code...

**Comment by OldRavenNL** [comment]
I love to see more support for tri's in general.
I'm currently using dRonin and although it flies great I miss some interesting features that iNav has.
(also flying one of rcexplorers Long range Tri's with a kakute F4 v2)

Since it's a long range Tri copter i really want to experiment more with gps functions and betaflight lacks those features.

It would be shame to see experimental projects like these Tri copters die because of the limited support.
It's projects like this that keep pushing interesting things foreward.

**Comment by zophon** [comment]
Hi,
Here to show support to have a iNav target for the Kakute F4 V2 on a tricopter configuration. This would be awesome. Having bought a tricopter LR recently, I would love to test and use this.

**Comment by MaxLansing** [comment]
I would like to second this request as well

**Comment by OldRavenNL** [comment]
dear mr stale

We are still testing this, we also have limited resources.

**Comment by OldRavenNL** [comment]
Still testing.

**Comment by BLeeKiis** [comment]
Is there a version that works with Matek F405Wing flight controller,please?
My FC FINALLY turned up in the mail- only been waiting since pre-Xmas sales! :/

**Comment by tgreer** [comment]
@digitalentity is there any chance of this getting pulled into the main branch?

**Comment by digitalentity** [comment]
It needs to be rebased on top of `development`.

**Comment by tgreer** [comment]
@jihlein Can you do the needful? It would be really good to see some nice tri support.

**Comment by jihlein** [comment]
Rebased to development.

**Comment by digitalentity** [comment]
It would also be good if you can add the new target to `release.mk`

**Comment by jihlein** [comment]
As luck would have it, while working on an unrelated project, I discovered that the GPIO pin used for the M4 output can be supported by a different timer resource than M3.  Making this change allows the existing target to support tricopters along with other multirotor configs.  I'm going to close this pull request and create another.

**Comment by digitalentity** [comment]
Superseded by #5684

**Pull request closed without merging**